docs/XMLReflection.md

```diff
@@ -57,6 +61,27 @@ A `<module>` element will contain 0 or more declarations of the types:
 - func
 - type declaration
 - property
+
+## Type Aliases
+
+Swift type aliases get aggregated into a list of `<typealias>` that live inside a `<typealiases>` tag.
+Typealiases are only needed when typealiases are present and the types presented have not been flattened to their actual type.
+The reflector flattens all aliased types, so it includes no typealiases. The `.swiftinterface` parser can't flatten typealiases, so it includes them.
+
+A `<typealias>` tag will contain the following attributes:
+
+- name - a string that represents the type being aliases including any generic part of its signature
+- accessibility - one of `Public`, `Private`, `Internal`, `Open`
+- type - a string that represents the target type to which name is aliased.
+For example,
+```
```

Comment:
I wonder if this would do syntax highlighting?

```suggestion
```swift
```

Reply:
I just checked. No. It will not. 
But ```Swift supposedly will.